components/ContactUs.jsx

```diff
@@ -2,12 +2,12 @@ const ContactForm = () => {
   const handleError = (e) => {
     e.target.setCustomValidity(
       'Du måste godkänna vår policy för att kunna skicka in ditt meddelande'
-    );
-  };
+    )
+  }
 
   const handleInput = (e) => {
-    e.target.setCustomValidity('');
-  };
+    e.target.setCustomValidity('')
+  }
```

Comment:
Seems like we have difference on trailing semi colons too, i'll remove them from mine

Reply:
Added the yarn.lock to git ignore